pages/listings/listing/index.html

```diff
@@ -4,54 +4,22 @@
-    <script src="https://kit.fontawesome.com/155738729b.js" crossorigin="anonymous"></script>
-
-    <!-- Page title of single listing will have to be rendered in JS to get correct info related to the listing -->
-    <title>PAge title</title>
+    <title id="singleListingPageTitle"><!-- Content rendered in JS--></title>
   </head>
   <body id="homePage">
     <header></header>
-    <main class="min">
+    <main class="min-vh-100">
       <div class="container">
-        <div class="row d-flex alig-items-center justify-content-center my-5">
-          <div class="col-lg-7 d-md-flex align-items-start">
-            <!-- Back button, need to add JS event to make it take you to previous page -->
-            <button class="btn border-0 p-1 me-3 my-2 my-md-0 fs-5"><i class="fa-solid fa-chevron-left"></i></button>
-
-            <div class="card bg-theme-light border-0">
-              <!-- Content will be moved to be rendered in JS, just hard coded for now -->
-              <img class="card-img-top" src="" alt="Placeholder" />
-              <div class="card-body d-flex flex-column gap-1 p-sm-3 p-md-5">
-                <h1 class="card-title">Headline title</h1>
-                <small>Job title <span>&#x2022;</span> Company name <span>&#x2022;</span> Location</small>
-                <small>1 day ago <span>&#x2022;</span> Deadline date <span>&#x2022;</span> 47 Applicants</small>
-                <div class="d-flex gap-3 my-3">
-                  <button class="btn btn-theme-primary text-uppercase text-white w-100">Apply Now</button>
-                  <button class="btn btn-theme-secondary"><i class="fa-regular fa-heart"></i></button>
-                </div>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Error, vel consequuntur. Perferendis minus quidem quia consequatur accusamus laudantium quos atque fuga quae, libero nemo aliquid excepturi in unde dolorem enim!</p>
-                <ul>
-                  <li>Lorem</li>
-                  <li>Lorem</li>
-                  <li>Lorem</li>
-                </ul>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum atque perspiciatis, explicabo inventore quo voluptatibus officiis numquam qui nulla illum impedit fuga perferendis et necessitatibus hic esse nostrum sit est.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium ut velit recusandae, perferendis quas quo quia impedit vitae nobis quod, illum quisquam tempora ipsa consequuntur est tenetur esse cumque sunt?</p>
-              </div>
-            </div>
-          </div>
+        <div class="row d-flex alig-items-center justify-content-center my-5" id="singleListingContainer">
+          <!-- Content rendered in JS -->
         </div>
       </div>
     </main>
     <footer></footer>
     <script src="/index.js" type="module"></script>
+    <script src="/src/js/index.js" type="module"></script>
```

Comment:
Is this and the line above the same?

Reply:
ah! yeah, okei, I'm unsure too, lets see what Åke says to this :)

Reply:
dont need that last script. 
Add it in the root index. -
like this: 
![image](https://user-images.githubusercontent.com/46559129/213176392-933e019d-07ce-4257-99ae-855cef66cb1b.png)


then to run it you will add it in the correct route case like this: 

```js
import { viewSingleListing } from './views/listings/index.js';
import * as ui from './src/js/ui/index.js';

ui.displayBaseLayout();



/**
 * 
 * Just a router switch do change page title and run the needed functions for each page.
 * The router switch is requiring a id on the body element of the html page. 
 * 
 * @example // Profile UI settings
                case 'dashboard':
                // Page title
                document.querySelector('title').innerText = defaultTitle + ` || ` + `Dashboard`;
                break;
 * 
 * 
 */
const routerSwitch = () => {
  // Default/Fallback page title
  const defaultTitle = 'Add default title';

  const routeName = document.body.id;
  switch (routeName) {
    // Homepage UI settings
    case 'homepage':
      // Page title
      document.querySelector('title').innerText = defaultTitle + ` || Homepage`;
      break;

    case '{{ ADD YOU BODY ID HERE }}':
      // Page title
      document.querySelector('title').innerText = defaultTitle + ` || Homepage`;
      viewSingleListing()
      break;

    // 404 UI settings
    case '404':
      // Page title
      document.querySelector('title').innerText = defaultTitle + ` || ` + '404';
      break;
    default:
  }
};
routerSwitch();